app/views/users/_additional_authentication.html.erb

```diff
@@ -1,17 +1,16 @@
-<% if SiteConfig.auth_allowed?("github") && !@user.identities.exists?(provider: "github") %>
-  <div class="field">
-    <a href="/users/auth/github" class="big-button cta" data-no-instant>
-      <img src="<%= asset_path("github-logo.svg") %>" alt="github logo"> CONNECT GITHUB ACCOUNT
-    </a>
+<% if SiteConfig.auth_allowed?("github") && SiteConfig.auth_allowed?("twitter") && (!@user.identities.exists?(provider: "github") || !@user.identities.exists?(provider: "twitter")) %>
```

Comment:
I think you changed the funtionality here, as the boolean conditions of before are different from the ones there are now. 

Before it said: "if github is enabled and the user hasn't already connected with github, show this section" (and separately from twitter, as they can be enabled separately)

Now it checks if both github and twitter are enabled, but either could be disabled

Reply:
hmm.. i'll double check. I think this is what I wanted:

```
<% if (SiteConfig.auth_allowed?("github") || SiteConfig.auth_allowed?("twitter")) && (!@user.identities.exists?(provider: "github") || !@user.identities.exists?(provider: "twitter")) %>
```

but i'll double check that.